packages/api/src/db/article.ts

```diff
@@ -22,6 +23,8 @@ export interface ArticleData {
   readonly socialMediaDescription?: string
   readonly socialMediaAuthorIDs: string[]
   readonly socialMediaImageID?: string
+
+  readonly comments: OptionalPublicComment[]
```

Comment:
I'm pretty sure you don't need this. The comments already reference the article, page or whatever through the `itemID`. Why would you want to put them into the Article?

Reply:
fixed